app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,92 @@
+.landing-page-body {
+  margin: 0 auto;
+  width: 100%;
+  padding-top: 0px !important;
+  min-height:100%;
+  background-color:#000000;
+  color:#eee;
+  overflow:hidden;
+}
+
+.v-center {
+  margin-top:7%;
+}
+
+/*
+ * Style tweaks
+ * --------------------------------------------------
+ */
+body {
+  padding-top: 50px;
+  background-color: #f5f5f5;
+}
+footer {
+  padding-left: 15px;
+  padding-right: 15px;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.